Library/Homebrew/requirements/ruby_requirement.rb

```diff
@@ -12,7 +12,20 @@ def initialize(tags)
     which_all("ruby").detect do |ruby|
       version = /\d\.\d/.match Utils.popen_read(ruby, "--version")
       next unless version
-      Version.create(version.to_s) >= Version.create(@version)
+      version = version.to_s
+      if version.to_f > 1.9
```

Comment:
Assuming that Ruby > 2.0 is installed, how would we access the portable-ruby package with brew? Could I just require rbconfig in ruby_requirements.rb?